dotcom-rendering/src/components/MiniProfile.tsx

```diff
@@ -0,0 +1,186 @@
+import { css } from '@emotion/react';
+import { type ArticleFormat } from '@guardian/libs';
+import { neutral, textSans14 } from '@guardian/source/foundations';
+import type { EditionId } from '../lib/edition';
+import type { ArticleElementRenderer } from '../lib/renderElement';
+import { slugify } from '../model/enhance-H2s';
+import { sanitiseHTML } from '../model/sanitise';
+import { palette } from '../palette';
+import type { ServerSideTests, Switches } from '../types/config';
+import type {
+	MiniProfile as MiniProfileModel,
+	StarRating,
+} from '../types/content';
+import { subheadingStyles } from './Subheading';
+
+const miniProfileStyles = css`
+	padding-top: 8px;
+`;
+
+const headingLineStyles = css`
+	width: 140px;
+	margin: 0 0 2px 0;
+	border: none;
+	border-top: 4px solid ${palette('--heading-line')};
+`;
+
+// Nesting is necessary in the bio styles because we receive a string of html from the
+// field. This can contain the following tags:
+// Blocks: p, ul, li
+// Inline: strong, em, a
+const bioStyles = css`
+	${textSans14};
+	padding-top: 6px;
+	color: ${palette('--mini-profiles-text-subdued')};
+	line-height: 1.3rem;
```

Comment:
Why is this necessary? Since the introduction of [specific text combinations like `textSans14`](https://github.com/guardian/csnx/pull/1335) we strongly discourage overriding line-height and other typography-related values.

Reply:
I'll avoid doing so, was working from a Figma file but happy to go in favour of consistency rather than design specifics that are probably flexible.